cargo-paradedb/src/bench_hits.rs

```diff
@@ -0,0 +1,345 @@
+        if run_cmd!(echo "$checksum $filename" | md5sum -c --status).is_ok() {
+            run_cmd!(echo "Dataset '$filename' already exists and is verified, skipping download...")?;
+            return Ok(());
+        } else {
+            run_cmd!(echo "Checksum mismatch. Re-downloading '$filename'...")?;
+        }
+    }
+
+    run_cmd!(echo "Downloading $filename dataset...")?;
+    run_cmd!(wget --no-verbose --no-check-certificate --continue -O $filename $url)?;
+    Ok(())
+}
+
+pub async fn bench_hits(url: &str, tag: &str, workload: &str) -> Result<()> {
+    let _os = run_fun!(uname)?;
+
+    println!("\n*********************************************************************************");
+    println!(
+        "* Benchmarking pg_lakehouse version '{}' against ClickBench",
+        tag
+    );
+    println!("*********************************************************************************\n");
+
+    let root_file_path = PathBuf::from_str("/tmp")?;
+    let single_file_path = root_file_path.join("hits.parquet").display().to_string();
+    let partitioned_dir_path = root_file_path.join("partitioned/").display().to_string();
+
+    if workload == "single" {
+        run_cmd!(echo "Using ClickBench's single Parquet file for benchmarking...")?;
+        download_and_verify(
+            "https://paradedb-benchmarks.s3.amazonaws.com/hits_5m_rows.parquet",
+            "5182ed3b0ad35137db38faac395d7f55",
+            &single_file_path,
+        )?;
+    } else if workload == "partitioned" {
+        run_cmd!(echo "Using ClickBench's one hundred partitioned Parquet files for benchmarking...")?;
+        run_cmd!(mkdir -p $partitioned_dir_path)?;
+        run_cmd!(
+           seq 0 99
+          | xargs "-P100" "-I{}" wget --no-verbose --no-check-certificate --directory-prefix $partitioned_dir_path
+            --continue "https://paradedb-benchmarks.s3.amazonaws.com/partitioned_5m_rows/hits_{}.parquet"
+        )?;
+    } else {
+        eprintln!("Invalid workload: {}", workload);
+        std::process::exit(1);
+    }
+
+    run_cmd!(echo "\nLoading dataset...")?;
+    env::set_var("PGPASSWORD", "postgres");
+
+    let conn_opts = &PgConnectOptions::from_str(url)?;
+    let mut conn = PgConnection::connect_with(conn_opts).await?;
+
+    let file_path = if workload == "single" {
+        single_file_path
+    } else {
+        partitioned_dir_path
+    };
+
+    println!("Creating foreign table from '{}'", file_path);
+
+    conn.execute(create_query_sql(&file_path).as_str())
+        .await
+        .unwrap();
+
+    run_cmd!(echo "\nRunning queries...")?;
+    run_queries(&mut conn).await?;
+
+    // run_cmd!(echo "\nPrinting disk usage...")?;
+    // if os.trim() == "Linux" {
+    //     run_cmd!(sudo docker exec paradedb du -bcs /bitnami/postgresql/data)?;
+    // } else {
+    //     run_cmd!(docker exec paradedb du -bcs /bitnami/postgresql/data)?;
+    // }
```

Comment:
You can leave this out of the benchmarks too, IMO. I can pick this up in the ClickBench repository where the Postgres being run will be just Ubuntu Postgres and we know which folder to read